File: rps.rb
#!/usr/bin/env ruby

# ah ruby how awesome
player = [:paper, :scissors, :rock][rand(3)]
computer = [:paper, :scissors, :rock][rand(3)]

# define rules

#Rock
if player == :rock && computer == :rock
  result = 'Draw'
end
if player == :rock && computer == :scissors
  result = 'Lose'
end
if player == :rock && computer == :paper
  result = 'Win'
end

#Paper
if player == :scissors && computer == :rock
  result = 'Win'
end
if player == :scissors && computer == :scissors
  result = 'Draw'
end
if player == :scissors && computer == :paper
  result = 'Lose'
end

#Scissors
if player == :paper && computer == :rock
  result = 'Lose'
end
if player == :paper && computer == :scissors
  result = 'Win'
end
if player == :paper && computer == :paper
  result = 'Draw'
end

print "\nPlayer : #{player}  \n"
print "Computer : #{computer} \n"
print "\nOutcome for the player : #{result}\n\n"

File: rps.php
<?php

$player = rand(1, 3);
$computer = rand(1, 3);

/*
1 = rock
2 = paper
3 = Scissor 
*/

echo 'Player: '.$player.'<br/>';
echo 'Computer: '.$computer.'<br/>';


if (($player == 1) && ($computer == 1)) {
	echo 'Draw';
}
if (($player == 1) && ($computer == 2)) {
	echo 'Lose';
}
if (($player == 1) && ($computer == 3)) {
	echo 'Win';
}

if (($player == 2) && ($computer == 1)) {
	echo 'Win';
}
if (($player == 2) && ($computer == 2)) {
	echo 'Draw';
}
if (($player == 2) && ($computer == 3)) {
	echo 'Lose';
}

if (($player == 3) && ($computer == 1)) {
	echo 'lose';
}
if (($player == 3) && ($computer == 2)) {
	echo 'Win';
}
if (($player == 3) && ($computer == 3)) {
	echo 'Draw';
}

/*
echo $hand_1;
echo $hand_2;
*/

?>